packages/fixie/src/isomorphic-client.ts

```diff
@@ -72,28 +73,24 @@ export class IsomorphicFixieClient {
     if (this.apiKey) {
       headers.Authorization = `Bearer ${this.apiKey}`;
     }
-
     if (debug) {
       console.log(`[Fixie request] ${this.url}${path}`, bodyData);
     }
-    const res = await fetch(`${this.url}${path}`, {
+    const res = await axios.request({
```

Comment:
This will introduce `axios` to the client bundle, which will massively increase the size of what devs need to accept in order to use `fixie/web`. This is not necessarily a dispositive consideration but I think it's worth being rigorous about what we're getting in exchange. 

Using https://minify-js.com/, I estimate the `IsomorphicClient` is ~3.5kb minified. By contrast, [axios is `~30kb`](https://bundlephobia.com/package/axios@1.5.1). So we're roughly 10x-ing the size of our bundle.

Axios was originally developed before native `fetch` was available. In the olden days, you'd definitely want to use it or `got`. Nowadays, I think it's less needed in general.

Reply:
Makes sense - will do this the old-fashioned way